staff/PacoGD/cv/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Curriculum Francisco Gala Doblas</h1> 
+    <h4>Experiencia laboral</h4>
+    <ul>
+        <li>Operario logistica Hama S.L.</li>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ratione unde error quisquam natus neque optio praesentium magnam dolor blanditiis! Blanditiis doloribus voluptatum, laudantium quaerat nemo sed sequi eum ut vero?</p>
+        <li>Operario logistica Decathlon España</li>
+    <h4>Idiomas</h4>
+        <ul>
+            <li>Castellano</li>
+            <li>Catalán</li>
+            <li>Inlgés</li>
+        </ul>
+
+        
+    </ul>
```

Comment:
Usually the structure is:
List 1:
< ul >   <------ unordered list
     < li >< /li >   <---- list item
     < li >< /li >
< /ul >

List 2
< ul >
     < li >< /li >
     < li >< /li >
< /ul >

Reply:
Caracelga